src/app/components_ngrx/planner-list/planner-list.component.html

```diff
@@ -20,7 +20,7 @@ <h4>Please wait, we are loading your data.</h4>
 </div>
 
 <ng-template #sidePanel>
-  <aside class="f8-sidepanel">
+  <aside class="f8-sidepanel" (scroll)="closePopover()">
```

Comment:
Why there is a scroll event here? There is one in the child section. Besides, why do you need to close the popovers on a scroll event?

Reply:
As discussed with @SMahil the popovers should be closed on scroll as it is a separate event like outside click. 
@SMahil ^^

Reply:
@sanbornsen one scroll event is for the side panel and another for the workitem list view